squad/frontend/templates/squad/compare_projects.jinja2

```diff
@@ -29,6 +29,16 @@
     </label>
   </div>
   {% endfor %}
+  <div class="form-group">
+    <label>{{ _('compare by') }}</label>
+    <div>
+      <label for="test-comparison">{{ _('tests') }}</label>
+      <input id="test-comparison" name="comparison_type" type="radio" value="test" checked="checked"/>
+      &nbsp;{{ _('or') }}&nbsp;
+      <label for="metric-comparison">{{ _('metrics') }}</label>
+      <input id="metric-comparison" name="comparison_type" type="radio" value="metric" />
+    </div>
+  </div>
```

Comment:
this does not work for internationalization; you should always use full sentences instead, e.g.
[ ] compare by tests
[ ] compare by metrics

when you break up the sentences, you assume that all languages have the same flow structure as English, and that is not the case.

Reply:
That's true, I didn't consider that into the equation. I've added a group label as "Comparison type", then added the options as you suggested. Looks much better indeed.